xclim/temperature.py

```diff
@@ -47,6 +47,23 @@ def cfprobe(self, da):
         checks.check_valid(da, 'standard_name', 'air_temperature')
 
 
+class TasminTasmax(BivariateIndicator):
```

Comment:
Build is failing. Looks as though the assignment was performing a necessary operation. Is it a matter of replacing the `BivariateIndicator` here with `Indicator`?

Reply:
Correct. Done.